packages/editor-ui/src/components/NodeSettings.vue

```diff
@@ -43,705 +39,369 @@ import { get, set, unset } from 'lodash-es';
 import type { EventBus } from 'n8n-design-system';
 import { useExternalHooks } from '@/composables/useExternalHooks';
 import { useNodeHelpers } from '@/composables/useNodeHelpers';
-import { importCurlEventBus, ndvEventBus } from '@/event-bus';
 import { useToast } from '@/composables/useToast';
+import { useI18n } from '@/composables/useI18n';
+import { useTelemetry } from '@/composables/useTelemetry';
+import { importCurlEventBus, ndvEventBus } from '@/event-bus';
 
-export default defineComponent({
-	name: 'NodeSettings',
-	components: {
-		NodeTitle,
-		NodeCredentials,
-		ParameterInputList,
-		NodeSettingsTabs,
-		NodeWebhooks,
-		NDVSubConnections,
-		NodeExecuteButton,
-	},
-	setup() {
-		const nodeHelpers = useNodeHelpers();
-		const externalHooks = useExternalHooks();
-		const { showMessage } = useToast();
-
-		return {
-			externalHooks,
-			nodeHelpers,
-			showMessage,
-		};
-	},
-	computed: {
-		...mapStores(
-			useHistoryStore,
-			useNodeTypesStore,
-			useNDVStore,
-			useUIStore,
-			useCredentialsStore,
-			useWorkflowsStore,
-			useWorkflowsEEStore,
-		),
-		isReadOnly(): boolean {
-			return this.readOnly || this.hasForeignCredential;
-		},
-		isExecutable(): boolean {
-			if (this.nodeType && this.node) {
-				const workflow = this.workflowsStore.getCurrentWorkflow();
-				const workflowNode = workflow.getNode(this.node.name);
-				const inputs = NodeHelpers.getNodeInputs(workflow, workflowNode!, this.nodeType);
-				const inputNames = NodeHelpers.getConnectionTypes(inputs);
-
-				if (!inputNames.includes(NodeConnectionType.Main) && !this.isTriggerNode) {
-					return false;
-				}
-			}
+const props = defineProps<{
+	eventBus: EventBus;
+	dragging: boolean;
+	pushRef: string;
+	nodeType: INodeTypeDescription | null;
+	readOnly?: boolean;
+	foreignCredentials: string[];
+	blockUI: boolean;
+	executable: boolean;
+	inputSize: number;
+}>();
+
+const emit = defineEmits<{
+	stopExecution: [];
+	redrawRequired: [];
+	valueChanged: [value: IUpdateInformation];
+	switchSelectedNode: [nodeTypeName: string];
+	openConnectionNodeCreator: [nodeTypeName: string, connectionType: NodeConnectionType];
+	activate: [];
+	execute: [];
+}>();
+
+const nodeTypesStore = useNodeTypesStore();
+const ndvStore = useNDVStore();
+const workflowsStore = useWorkflowsStore();
+const credentialsStore = useCredentialsStore();
+const historyStore = useHistoryStore();
+
+const telemetry = useTelemetry();
+const nodeHelpers = useNodeHelpers();
+const externalHooks = useExternalHooks();
+const i18n = useI18n();
+const { showMessage } = useToast();
+
+const nodeValid = ref(true);
+const openPanel = ref<'params' | 'settings'>('params');
+const nodeValues = ref<INodeParameters>({
+	color: '#ff0000',
+	alwaysOutputData: false,
+	executeOnce: false,
+	notesInFlow: false,
+	onError: 'stopWorkflow',
+	retryOnFail: false,
+	maxTries: 3,
+	waitBetweenTries: 1000,
+	notes: '',
+	parameters: {},
+});
+const nodeValuesInitialized = ref(false);
```

Comment:
Was this dropped intentionally?

```suggestion
// Used to prevent nodeValues from being overwritten by defaults on reopening ndv
const nodeValuesInitialized = ref(false);
```